index.html

```diff
@@ -2056,12 +2043,20 @@ <h2>The <dfn>record type</dfn> string</h2>
       <th>Description (<em>non-normative</em>)</th>
      </tr>
     </thead>
-    <tbody data-link-for="NDEFWriter">
+    <tbody data-link-for="NDEFReader">
+     <tr>
+      <tr>
+        <td><dfn>[[\ScanAbortSignal]]</dfn></td>
+        <td>`undefined`</td>
+        <td>
+          The {{NDEFScanOptions}}.signal to abort the operation.
```

Comment:
Shall we specify it's only about the scan operation?

Reply:
Done